error-handling-avro/src/test/java/com/bakdata/kafka/AvroDeadLetterProcessorTest.java

```diff
@@ -88,11 +89,12 @@ protected Properties getKafkaProperties() {
 
     @Test
     void shouldConvertAndSerializeAvroDeadLetter() {
+        final long startTimestamp = System.currentTimeMillis();
```

Comment:
Is there an option to access wallclock time via TestDriver? Because TestDriver has advanceWallclockTime. But maybe that is just used for punctuates

Reply:
I looked into, and I don't really see an easy way to access it